src/dag.rs

```diff
@@ -338,17 +314,11 @@ mod tests {
             HashSet::from_iter(order.clone().map(|node| node.id)),
             "Not all nodes visited"
```

Comment:
Is the instability mentioned in the comment above still present? Or could we just hard-code the `want` values now?

Reply:
Ah no you're right, I hard-coded the expected output.